ui/app/mainui/activitycenter/controls/Badge.qml

```diff
@@ -8,8 +8,9 @@ import StatusQ.Core.Theme 0.1
 Rectangle {
     id: root
 
-    height: visible ? 24 : 0
-    width: childrenRect.width + Style.current.smallPadding * 2
+    z: 100 // NOTE: workaround for message overlay
+    implicitWidth: childrenRect.width + Style.current.smallPadding
```

Comment:
Shouldn't padding be multiplied by 2?

Reply:
No, because border is half of circle: 2 / 2 = 1 :)